src/lib.rs

```diff
@@ -107,21 +112,21 @@ struct CommonOptions {
     #[structopt(long)]
     enable_simd: bool,
 
-    /// Enable support for reference types
+    /// Disable support for reference types
     #[structopt(long)]
-    enable_reference_types: Option<bool>,
+    disable_reference_types: bool,
```

Comment:
FWIW I was just thinking about this recently and how we might have to do this. It seems kinda bad that we're guaranteed to break anyone using these flags as soon as things stabilize, and it feels kinda odd having a mishmash of enable/disable.

To help reduce the clutter in the CLI options as well as to make this a bit more future proof, what do you think of something like:

```
$ wasmtime --wasm-features=+simd
$ wasmtime --wasm-features=-bulk-memory
$ wasmtime --wasm-features=+multi-value,-bulk-memory,+threads
$ wasmtime --wasm-features=+all
```

(similar to LLVM features and `-Ctarget-feature` in Rust)

This also seems fine for me to file a follow-up too

Reply:
I've added a `--wasm-features` option with the latest commit.